korlibs-crypto/src/korlibs/memory/InternalCryptoArrays.kt

```diff
@@ -0,0 +1,11 @@
+package korlibs.memory
+
+/** Copies [size] elements of [src] starting at [srcPos] into [dst] at [dstPos]  */
+internal fun internalArrayCopy(src: ByteArray, srcPos: Int, dst: ByteArray, dstPos: Int, size: Int) {
```

Comment:
Can you keep the `arraycopy` name? and move this file to the `korlibs.memory.internal` package?

Reply:
Fixed